cli/units_cmd.go

```diff
@@ -111,6 +118,22 @@ func scanUnitsIntoConfig(cfg *config.Repository, quiet bool) error {
 	return nil
 }
 
+// checkRegularExecutableFile verifies that the file pointed to by name is a regular executable file.
+func checkRegularExecutableFile(name string) error {
+	reason := error(nil)
```

Comment:
A more idiomatic way to express this is:

```Go
var reason error
```

The zero value for error type is already nil.

(For reference, see http://dave.cheney.net/2014/05/24/on-declaring-variables.)